WordPress/Classes/Services/BlogService.m

```diff
@@ -159,9 +159,14 @@ - (void)syncBlogsForAccount:(WPAccount *)account
 {
     DDLogMethod();
 
+    NSManagedObjectID *accountID = account.objectID;
     id<AccountServiceRemote> remote = [self remoteForAccount:account];
     [remote getBlogsWithSuccess:^(NSArray *blogs) {
         [self.managedObjectContext performBlock:^{
+            WPAccount *account = (WPAccount *)[self.managedObjectContext existingObjectWithID:accountID error:nil];
+            if (!account) {
+                return;
```

Comment:
That's another bug then. I could reproduce and it was caused because the account wasn't saved yet. Calling `obtainPermanentIDsForObjects:error:` fixes it